railties/lib/rails/commands/commands_tasks.rb

```diff
@@ -151,31 +172,11 @@ def write_help_message
         puts HELP_MESSAGE
       end
 
-      # Output an error message stating that the attempted command is not a valid rails command.
-      # Run the attempted command as a rake command with the --dry-run flag. If successful, suggest
-      # to the user that they possibly meant to run the given rails command as a rake command.
-      # Append the help message.
-      #
-      #   Example:
-      #   $ rails db:migrate
-      #   Error: Command 'db:migrate' not recognized
-      #   Did you mean: `$ rake db:migrate` ?
-      #   (Help message output)
-      #
-      def write_error_message(command)
-        puts "Error: Command '#{command}' not recognized"
-        if %x{rake #{command} --dry-run 2>&1 } && $?.success?
-          puts "Did you mean: `$ rake #{command}` ?\n\n"
-        end
-        write_help_message
-        exit(1)
-      end
-
       def parse_command(command)
         case command
-        when '--version', '-v'
+        when '--version', '-v', 'version'
           'version'
-        when '--help', '-h'
+        when '--help', '-h', 'help'
```

Comment:
No need to add `'help'` or `'version'`, they would just hit the `else` branch here, which would let them through just the same.